poetry.lock

```diff
@@ -324,7 +324,7 @@ docs = ["furo", "jaraco.packaging (>=9)", "jaraco.tidelift (>=1.4)", "rst.linker
 testing = ["flake8 (<5)", "pytest (>=6)", "pytest-black (>=0.3.7)", "pytest-checkdocs (>=2.4)", "pytest-cov", "pytest-enabler (>=1.3)", "pytest-flake8", "pytest-mypy (>=0.9.1)"]
 
 [[package]]
-name = "Levenshtein"
+name = "levenshtein"
```

Comment:
Do you know why the name changed here? Did you previously edit this manually?

Reply:
https://pypi.org/project/Levenshtein/ 

For some reason, both `pip install Levenshtein` and `pip install levenshtein` are present on this page. I remember reading somewhere that case for PyPI _shouldn't_ matter except for some specific cases.

Reply:
Yes. This was previously edited manually, but changes in API last week (https://github.com/phylum-dev/api/pull/613 and https://github.com/phylum-dev/api/pull/616) fixed this so package name casing no longer matters.

https://github.com/phylum-dev/api/issues/612 is the issue that remains to account for the Python distribution/package names that include `.`, `-`, and `_` characters.